kinematics_base_test/src/test_kinematics_plugin.cpp

```diff
@@ -167,6 +168,10 @@ class KinematicsTest
   int num_ik_tests_;
   int num_ik_multiple_tests_;
   int num_nearest_ik_tests_;
+
+  // random generator
+  random_numbers::RandomNumberGenerator rng_{RANDOM_SEED};
```

Comment:
is this aggregate initialization? im not used to this syntax http://en.cppreference.com/w/cpp/language/aggregate_initialization

Reply:
it is C++11 Brace-Initialization, can be used also for class members. check the link http://www.informit.com/articles/article.aspx?p=1852519